### PR TITLE
Plink 1.07 x86 65 goolf 1.7.20

### PR DESCRIPTION
**Pull request by npklein**

### CODE REVIEW

**Comment by boegel** [comment]
Jenkins: ok to test

File: easybuild/easyconfigs/p/PLINK/PLINK-1.07-x86_64-goolf-1.7.20.eb [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+description = 'plink-1.07-src: Whole-genome association analysis toolset'
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
+
+sources = ['%(namelower)s-%(version)s.zip']
+source_urls = ['http://pngu.mgh.harvard.edu/~purcell/plink/dist/']
+
+checksums = ['fd0bafeda42151b20534e4f97b0d97df']
+
+files_to_copy = [
```
Comment (boegel) [inline easybuild/easyconfigs/p/PLINK/PLINK-1.07-x86_64-goolf-1.7.20.eb]: for a simple list like this, it's OK to just do it single-line

but you can leave this as-is as well, up to you

Comment (npklein) [inline easybuild/easyconfigs/p/PLINK/PLINK-1.07-x86_64-goolf-1.7.20.eb]: The other already existing PLINK .eb files do it multiple lines, is it okay if they differ in style?

Comment (boegel) [inline easybuild/easyconfigs/p/PLINK/PLINK-1.07-x86_64-goolf-1.7.20.eb]: Ah, in that case, keep it as is, it's better to have similar style across versions, indeed.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
Linux RHEL 6.6, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/8e387a033437f05ec129 for a full test report.

**Comment by boegel** [comment]
Going in, thanks @npklein!

**Pull request merged**